templates/create-daml-app/ui/src/serviceWorker.ts

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
If you remove service workers, we can kill this whole file.

Reply:
Deleted. FYI @hurryabit